client/jetpack-connect/jetpack-connect-notices.jsx

```diff
@@ -161,11 +183,11 @@ class JetpackConnectNotices extends Component {
 	}
 
 	render() {
-		const values = this.getNoticeValues();
-		if ( values ) {
+		const noticeValues = this.getNoticeValues();
+		if ( noticeValues ) {
```

Comment:
I'd imported `values`, but ended up not needing it. This was replaced to avoid shadowing, but can be reverted.

Reply:
Totally fine.